### DISCUSSION TITLE
@Html.Raw(DynamicScriptManager.GetScriptText("Lookup.Administration.Language"))

### DISCUSSION THREAD

**Discussion by mahedi2810hasan** [serenity-is/Serenity#5302] [Help]
I have created a project without selecting any features and modules. Then change the connectionString to my local database & then when I try to run my project the first time, it's always showing this error.

How can I solve it? Please help me.

**Comment by VictorTomaili** [comment]
Can you give more details? Which version of serene do you use, what's the error message (screenshots are welcome) etc.
